### PR TITLE
uses bound drop component to select difficulty

### PR DESCRIPTION
**Pull request by willemolding**
This could probably do with some improvement but the general framework is there

### CODE REVIEW

**Comment by willemolding** [comment]
Propose using different names for the difficulty settings. Possibly something funny and crypto related?

File: ui-src/src/components/CreateGameForm.tsx [hunk 1]
```diff
@@ -7,20 +7,25 @@ import * as actions from '../actions';
 import { fetchJSON } from '../common';
 import store from '../store';
 
+const EASY_DENSITY: number = 0.1;
+const INTERMEDIATE_DENSITY: number = 0.3;
+const EXPERT_DENSITY: number = 0.5;
```
Comment (maackle) [inline ui-src/src/components/CreateGameForm.tsx]: pretty sure 0.5 is wayyyy too much

File: ui-src/src/components/CreateGameForm.tsx [hunk 2]
```diff
@@ -70,6 +94,11 @@ class CreateGameForm extends React.Component<any, CreateGameFormState> {
         })
     }
   }
+
+  private handleChange(event) {
+    console.log(event);
```
Comment (maackle) [inline ui-src/src/components/CreateGameForm.tsx]: could remove

**Review by maackle — APPROVED** [review]
Looks good, could address 2 things I mentioned though

**Pull request merged**